<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>605</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1086</width>
    <height>605</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1086</width>
    <height>605</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Wakxy Sniffer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonProxy">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>530</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Démarrer le proxy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCapture">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>530</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Commencer la capture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSendPacket">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Envoyer un packet modifié</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxSnifferType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Type de sniffing</string>
    </property>
    <widget class="QRadioButton" name="radioButtonAuthSniffing">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Authentification</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonWorldSniffing">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>En jeu</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="textBrowserLog">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>1061</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetPacket">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>1061</width>
      <height>221</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>#</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Type</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Size</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Opcode</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ASCII</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Hex</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="labelServer">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>40</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>#########</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonReloadConf">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>530</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Recharger le fichier de configuration</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1086</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionOuvrir"/>
    <addaction name="actionSauvegarder"/>
    <addaction name="actionQuitter"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOuvrir">
   <property name="text">
    <string>Ouvrir</string>
   </property>
  </action>
  <action name="actionSauvegarder">
   <property name="text">
    <string>Sauvegarder</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
